src/Microsoft.AspNetCore.Localization/CookieRequestCultureProvider.cs

```diff
@@ -118,5 +145,8 @@ public static ProviderCultureResult ParseCookieValue(string value)
 
             return new ProviderCultureResult(cultureName, uiCultureName);
         }
+
+        private static bool IsValidCulture(string cultureName) =>
+            CultureInfo.GetCultures(CultureTypes.SpecificCultures).Any(c => c.Name == cultureName);
```

Comment:
Believe it or not I was thinking about that yesterday, so that's why I wrote my comment [above](https://github.com/aspnet/Localization/pull/385#discussion_r134286514)

I will fix it now ..